frontend/src/components/explorer/mapViewer/SidebarDataPanels.vue

```diff
@@ -12,69 +12,12 @@
-              are missing as the {{ currentMap.type }} is being updated much more often than the maps.
-              Also, as the maps are manually curated, occasionally some reactions cannot be added.
-              The number of reactions shown is {{ mapReactionList.length }}.
-            </p>
-            <p v-else class="pb-4">
-              There are {{ missingReactionList.length }} reactions not shown on any of the {{ currentMap.name }} maps.
-              The 2D map of {{ currentMap.name }} is split into multiple maps due to its size. The number of missing
-              reactions displayed is the missing reactions when the reactions in all {{ currentMap.name }} maps are
-              combined.
-              Some reactions are missing as the {{ currentMap.type }} is being updated much more often than the maps.
-              Also, as the maps are manually curated, occasionally some reactions cannot be added.
-              The number of reactions shown is {{ mapReactionList.length }}.
-            </p>
-            <table class="table main-table is-fullwidth m-0">
-              <tbody>
-                <tr>
-                  <td class="td-key has-background-primary has-text-white-bis">
-                    {{ currentMap.mapReactionIdSet.length == 1 ? "Missing reactions on the map" : `Missing reactions
-                    on the combined ${currentMap.name} maps` }}
-                  </td>
-                  <td>
-                    <div v-html="missingReactionIdListHtml"></div>
-                    <div v-if="!showFullReactionListMissing && missingReactionList.length > displayedReaction">
-                      <br>
-                      <button class="is-small button" @click="showFullReactionListMissing=true">
-                        ... and {{ missingReactionList.length - displayedReaction }} more
-                      </button>
-                    </div>
-                  </td>
-                </tr>
-                <tr>
-                  <td class="td-key has-background-primary has-text-white-bis">
-                    {{ currentMap.mapReactionIdSet.length == 1 ? "Reactions shown on the map" : `Reactions shown
-                    on the combined ${currentMap.name} maps` }}
-                  </td>
-                  <td>
-                    <div v-html="mapReactionIdListHtml"></div>
-                    <div v-if="!showFullReactionListMap && mapReactionList.length > displayedReaction">
-                      <br>
-                      <button class="is-small button" @click="showFullReactionListMap=true">
-                        ... and {{ mapReactionList.length - displayedReaction }} more
-                      </button>
-                    </div>
-                  </td>
-                </tr>
-              </tbody>
-            </table>
-          </div>
-          <button class="modal-close is-large" @click="closeModal"></button>
+          <a @click="$emit('update:showModal', true)"> See comparison </a>
```

Comment:
Minor thing and I don't think it was exactly introduced in this PR but we can avoid a bit more repetition with the following.

```suggestion
          <template v-if="currentMap.mapReactionIdSet.length == 1">
            Please note that {{ missingReactionList.length }}
            of the reactions in the {{ currentMap.type }} are not shown on the map.
          </template>
          <template v-else>
            Please note that {{ missingReactionList.length }} of the reactions in the
            {{ currentMap.type }} are not shown on any of the {{ currentMap.name }} maps.
          </template>
          <a @click="$emit('update:showModal', true)"> See comparison </a>
```

Reply:
Nice suggestion! Fixed with 95d825942bddc73f0a804e0751727527fa0e85db